spec/controllers/api/v1/monsters_controller_spec.rb

```diff
@@ -41,13 +41,36 @@
       expect(response.status).to eq 200
       expect(response.content_type).to eq("application/json; charset=utf-8")
     end
+    it "should not accept a monster without the right attributes" do
+      post_json = {
+        monster: {
+          name: "",
+          description: "It's pretty self explanatory",
+          classification: "Zombie",
+          habitat: "City"
+        }
+      }
+      post_json = {
+        monster: {
+          name: "Frankenstein's Monster",
+          description: "",
+          classification: "Zombie",
+          habitat: "City"
+        }
+      }
```

Comment:
`post_json` is defined twice here, the second one is overwritting the first! You could combine the 2 test cases and try to `post` an empty monster